### PR TITLE
Updates SCC to version 5.4.2

### PR DESCRIPTION
**Pull request by eemperor**

### CODE REVIEW

File: dev/files-repo/terraform.tfvars [hunk 1]
```diff
@@ -16,8 +16,8 @@ uri_map = {
   "https://repo.saltstack.com/windows/Salt-Minion-3003.3-Py3-AMD64-Setup.exe"           = "saltstack/salt/windows/"
 
   # spawar scc
-  "s3://wrangler-watchmaker-filecache/spawar/scc/SCC_5.4.1_Windows_Setup.exe" = "spawar/scc/"
-  "s3://wrangler-watchmaker-filecache/spawar/scc/scc-5.4.1.rhel7.x86_64.rpm"  = "spawar/scc/"
+  "s3://wrangler-watchmaker-filecache/spawar/scc/SCC_5.4.2_Windows_Setup.exe" = "spawar/scc/"
```
Comment (lorengordon) [inline dev/files-repo/terraform.tfvars:19]: We aren't using this version yet right? So removing the old version won't have any impact?

**Review by lorengordon — APPROVED** [review]
Approving, presuming we aren't using v5.4.1...

File: dev/files-repo/terraform.tfvars (hunk 1 above)
Comment (eemperor) [reply dev/files-repo/terraform.tfvars:19]: Right.  It was only for testing.  Do you want me to just remove it from the history?

Comment (eemperor) [reply dev/files-repo/terraform.tfvars:19]: Nevermind.  I'm thinking of something else...

Comment (lorengordon) [reply dev/files-repo/terraform.tfvars:19]: Nah, ready to go, merge away

**Pull request merged**